caphandler/subscribeEvents.go

```diff
@@ -235,7 +233,7 @@ func getDeviceDetails(ctx iris.Context) (*caputilities.RedfishDevice, *capmodel.
 	//Get device details from request
 	err := ctx.ReadJSON(&deviceDetails)
 	if err != nil {
-		log.Println("Error while trying to collect data from request: ", err)
+		log.Error("while trying to collect data from request, PluginCiscoACI got: " + err.Error())
 		ctx.StatusCode(http.StatusBadRequest)
 		ctx.WriteString("Error: bad request.")
```

Comment:
logging some error and writing some other error. please write the same error

Reply:
Addressed